internal/config/config.go

```diff
@@ -107,6 +107,13 @@ var config struct {
 		Heartbeat   time.Duration `env:"HEARTBEAT" env-default:"30s" env-description:"heartbeat interval (time interval syntax)"`
 		MaxBeats    int           `env:"MAX_BEATS" env-default:"10" env-description:"maximum amount of heartbeats allowed"`
 	} `env-prefix:"WORKER_"`
+	Unleash struct {
+		Enabled     bool   `env:"ENABLED" env-default:"false" env-description:"unleash service (feature flags)"`
+		Environment string `env:"ENVIRONMENT" env-default:"" env-description:"unleash environment"`
+		Prefix      string `env:"PREFIX" env-default:"app.provisioning" env-description:"unleash flag prefix"`
+		URL         string `env:"URL" env-default:"http://localhost:4242" env-description:"unleash service URL"`
+		Token       string `env:"TOKEN" env-default:"" env-description:"unleash service client access token"`
+	} `env-prefix:"UNLEASH_"`
```

Comment:
I also added environment, by default it is unused and we will likely not need it (only if we share a single instance across environments which is not the case I think).